src/components/ShareLink.js

```diff
@@ -8,26 +8,25 @@ import LText from "./LText";
 import colors from "../colors";
 
 type Props = {
-  style?: *,
-  children: string | React$Element<*>,
-  string: string, // String to be shared
+  children: string | React$Node,
```

Comment:
React$Node includes `string` in it so you can just set the whole type as `React$Node`

Reply:
shit... will change